frontend/.env.sample

```diff
@@ -0,0 +1 @@
+API_URL = http://192.168.160.76:3000
```

Comment:
lets call it `API_ENDPOINT` it makes more sense

Reply:
ok